app/assets/stylesheets/emition_entry.scss

```diff
@@ -0,0 +1,133 @@
+@import "colors";
+@import "design_system_vibereport";
+
+.emotion-entry__h1{
```

Comment:
I think this class name should look like emotion-entry with parent tag h1:
h1 {
.emotion-entry {.....}
}

no point to make difficult class names here